<ui version="4.0" >
 <class>FixtureEditor</class>
 <widget class="QWidget" name="FixtureEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Fixture</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="m_tab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="General" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>568</width>
        <height>498</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="m_manufacturerLabel" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Manufacturer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_manufacturerEdit" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="m_modelLabel" >
           <property name="text" >
            <string>Model</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_modelEdit" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="m_typeLabel" >
           <property name="text" >
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_typeCombo" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text" >
             <string>Color Changer</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Dimmer</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Effect</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Fan</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Flower</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Hazer</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Laser</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Moving Head</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Other</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Scanner</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Smoke</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Strobe</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Channels" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>568</width>
        <height>498</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Channels</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QTreeWidget" name="m_channelList" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick" >
          <bool>false</bool>
         </property>
         <column>
          <property name="text" >
           <string>Channel Name</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Channel Group</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QToolButton" name="m_addChannelButton" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="iconSize" >
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_removeChannelButton" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="iconSize" >
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_editChannelButton" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="iconSize" >
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Modes" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>568</width>
        <height>498</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Modes</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QTreeWidget" name="m_modeList" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <property name="expandsOnDoubleClick" >
          <bool>false</bool>
         </property>
         <column>
          <property name="text" >
           <string>Mode Name</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Number of Channels</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QToolButton" name="m_addModeButton" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="iconSize" >
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_removeModeButton" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="iconSize" >
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_editModeButton" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="iconSize" >
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
